src/main/java/org/craftercms/studio/api/v2/utils/GitRepositoryHelper.java

```diff
@@ -697,25 +713,8 @@ private boolean removePublishBlackList(Repository publishedRepo) {
 
         List<String> patterns = Arrays.asList(StringUtils.split(blacklistConfig, STRING_SEPARATOR));
         try (Git git = new Git(publishedRepo)) {
-            String rootPath = publishedRepo.getWorkTree().getPath();
-            RmCommand rmCommand = git.rm();
-            Files.walkFileTree(Paths.get(rootPath), new SimpleFileVisitor<>() {
-                @Override
-                public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
-                    String sitePath = file.toAbsolutePath().toString().replaceFirst(rootPath, FILE_SEPARATOR);
-                    boolean isMatched = ContentUtils.matchesPatterns(sitePath, patterns);
-                    if (isMatched) {
-                        String gitPath = getGitPath(sitePath);
-                        rmCommand.addFilepattern(gitPath);
-                    }
-                    return FileVisitResult.CONTINUE;
-                }
-            });
-            retryingRepositoryOperationFacade.call(rmCommand);
+            // TODO: implement this for the new publishing system with published repo being bare
```

Comment:
Is this done already?

Reply:
It is not. There is a task in the ticket to revisit this.